src/components/common.js

```diff
@@ -579,7 +581,12 @@ export const DeleteConfirmationModal = ({ title, children, confirmationPrompt =
       onClick: del,
       disabled: !isConfirmed,
       tooltip: isConfirmed ? undefined : 'You must type the confirmation message'
-    }, buttonText)
+    }, buttonText),
+    onAfterOpen: () => {
```

Comment:
I tried `autoFocus` but it did not work. I don't exactly know why, but I suspect there's an issue with focus management when modals are opened from popups.

Reply:
^ Seems to be a better solution. Without the setTimeout, I'm seeing that focus does not return to the column menu after the dialog is dismissed… it seems to go nowhere. To see this, tab to the column menu, trigger the delete or clear choice, tab to Cancel, and dismiss the dialog.

It might be worth adding documentation to ConfirmationModal explaining the need for setTimeout from popup menus for proper focus behavior.

Reply:
Nice catch. Do you think we should add a `setTimeout` wrapper around `onClick` in the `MenuButton` component to handle this everywhere? At least in the data tab, there are several popup menus that open modals.